src/service/memberHandler/postMember.jsx

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+import { appSettings } from "../../Appdata/appdata";
+import { toast } from "react-toastify";
+import deleteImageHandler from "../deleteImageService";
+
+export default function submitMember(memberData, profileURL) {
```

Comment:
We only need the memberData. 

profileURL is not needed since this data is stored already inside memberData.

Reply:
This is for FB Storage Deletion marlon, incase error ang response sa server